healthrex_ml/extractors/starr_extractors.py

```diff
@@ -505,6 +574,87 @@ def __call__(self):
         query_job = self.client.query(query)
         query_job.result()
 
+class LabOrderNumberExtractor():
```

Comment:
This is great!  I'm wondering if this can be made to extend to more use cases if instead of having a new Extractor class specific to grabbing CBC and Metabolic panel orders, we add an optional list of `description` names to include in the final result of the query as an argument in the __init__() method of the LabOrderExtractor definition. What do you think?

Basically add new argument to __init__() of LabOrderExtractor called `include_list` that defaults to None. When it's not none we only include labs in that list.